doc/state_estimation_nodes.rst

```diff
@@ -62,6 +62,9 @@ The default values for ``map_frame``, ``odom_frame``, and ``base_link_frame`` ar
 ~transform_time_offset
 ^^^^^^^^^^^^^^^^^^^^^^
 Some packages require that your transforms are future-dated by a small time offset. The value of this parameter will be added to the timestamp of *map->odom* or *odom->base_link* transform being generated by the state estimation nodes in ``robot_localization``.
+~transform_timeout
```

Comment:
If we can fix this documentation issue, I can test and merge this. No rush at all, though.